### PR TITLE
Adding OCP doc

### PR DESCRIPTION
**Pull request by pburgisser**
Adding OCP documentation for IPI installation
Adding example for OCP IPI on AWS

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ** xref:ROOT:quickstart.adoc[Quickstarts]
 *** xref:ROOT:howtos/quickstart_eks.adoc[EKS]
 *** xref:ROOT:howtos/quickstart_aks.adoc[AKS]
+*** xref:ROOT:howtos/quickstart_ocp_ipi.adoc[OpenShift (IPI)]
```
Comment (mcanevet) [inline docs/modules/ROOT/nav.adoc:8]: Is it really OCP IPI or OCP IPI on AWS?

File: examples/ocp-ipi-aws/terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+locals {
+  install_config_path = "install-config.yaml"
+  region              = "eu-west-1"
+  base_domain         = "example.com"
+  cluster_name        = "ocp"  
+}
+provider "aws" {
```
Comment (mcanevet) [inline examples/ocp-ipi-aws/terraform/main.tf:7]: Please keep 1 space between blocks

File: examples/ocp-ipi-aws/terraform/main.tf [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  region = local.region
+}
+
+
```
Comment (mcanevet) [inline examples/ocp-ipi-aws/terraform/main.tf:11]: And only one space

**Comment by pburgisser** [comment]
@raphink Can you have a look please?

File: docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+:terraform-provider: AWS
+:terraform-provider-link: https://registry.terraform.io/providers/hashicorp/aws/latest/docs
+
+IMPORTANT: The example given below is made to work on AWS, other cloud providers such as Azure will come later.
+
+== Prerequisites
+
+- Access to API keys allowing to create required resources in AWS,
+- Meet the requirements for OCP deployment such as SSH public key and https://cloud.redhat.com[PullSecret] from Red Hat,
+- Binary https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest[openshift-install] in your $PATH,
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc:12]:
```suggestion
- The https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest[openshift-install] binary is in your $PATH,
```

File: docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+- Binary https://aws.amazon.com/cli/[AWS cli] in your $PATH,
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc:13]:
```suggestion
- The https://aws.amazon.com/cli/[AWS cli] binary is in your $PATH,
```

File: docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+include::partial$pipeline_outputs_ocp.adoc[]
+
+include::partial$terraform_backend.adoc[]
+
+include::partial$ocp_ipi_instant_intro.adoc[]
+
+include::partial$deploy_workstation.adoc[]
+
+== Follow deployment process
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc:48]:
```suggestion
== Deployment Process
```

File: docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+OpenShift is deployed using a nested terraform where a specific version is packaged within the openshift-install binary. When DevOps Stack terraform is running, we cannot follow the child process.
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc:50]:
```suggestion
OpenShift is deployed using the `openshift-install` binary, which embarks a nested Terraform version. When the DevOps Stack Terraform is running, we cannot follow the child process.
```

File: docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+Thus, we can track the deployment in terraform/<CLUSTER NAME>/.openshift_install.log
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_ocp_ipi_aws.adoc:52]:
```suggestion
Thus, we can track the deployment progress in `terraform/<CLUSTER NAME>/.openshift_install.log`
```

File: docs/modules/ROOT/partials/ocp_ipi_instant_intro.adoc [hunk 9]
```diff
@@ -0,0 +1,38 @@
+== Create your IPI response file
+
+The install-config.yaml file required to perform silent install of OpenShift
```
Comment (raphink) [inline docs/modules/ROOT/partials/ocp_ipi_instant_intro.adoc:3]:
```suggestion
Create the `install-config.yaml` file required to perform silent install of OpenShift:
```

File: docs/modules/ROOT/partials/ocp_ipi_instant_intro.adoc [hunk 10]
```diff
@@ -0,0 +1,38 @@
+== Create your IPI response file
```
Comment (raphink) [inline docs/modules/ROOT/partials/ocp_ipi_instant_intro.adoc:1]:
```suggestion
== Create your IPI Response File
```

File: docs/modules/ROOT/partials/ocp_kubeconfig.adoc [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+$ terraform output -json kubeconfig | jq -r .
+```
+
+or
+
+```shell
+$ export KUBECONFIG=<CLUSTER NAME>/auth/kubeconfig
+```
+
+Then, you should be able to use `oc`.
```
Comment (raphink) [inline docs/modules/ROOT/partials/ocp_kubeconfig.adoc:15]:
```suggestion
You should then be able to use `oc`.
```

File: docs/modules/ROOT/partials/ocp_kubeconfig.adoc [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+
+== Get the kubeadmin password
```
Comment (raphink) [inline docs/modules/ROOT/partials/ocp_kubeconfig.adoc:17]:
```suggestion
== Get the Kubeadmin Password
```

**Review by raphink — APPROVED** [review]

**Pull request merged**